pulsar-websocket/src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java

```diff
@@ -18,27 +18,21 @@
  */
 package org.apache.pulsar.websocket.stats;
 
+import static io.netty.util.internal.PlatformDependent.maxDirectMemory;
```

Comment:
This import is illegal based on the rules:
```
```
➜  sn-pulsar git:(neng/pulsar-module-checkstyle) mvn checkstyle:check -Dpulsar.basedir=. -pl pulsar-websocket
...
[ERROR] src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java:[21,1] (imports) IllegalImport: Illegal import - io.netty.util.internal.PlatformDependent.maxDirectMemory.
```

But I don't have a good idea on how to fix this one. Need some help on it.

Reply:
https://github.com/apache/pulsar/blob/f25b4a520bb82c6a09d373c9d314ca8e55d1f57b/buildtools/src/main/resources/pulsar/checkstyle.xml#L135-L137

we can edit the rule in my opinion

Reply:
@nlu90 We can make a suppression on `PlatformDependent.maxDirectMemory`, it's useful and widely use, like
```java
// CHECKSTYLE.OFF: IllegalImport
import io.netty.util.internal.PlatformDependent;
// CHECKSTYLE.ON: IllegalImport
```